tests/cortex/test_cortex_help_messages_312.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2024 Snowflake Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+import pytest
+
+if sys.version_info < (3, 12):
+    pytest.skip(
```

Comment:
Why not `@pytest.mark.skipif`?

Reply:
I want to avoid calling the whole file. If the file is called, all unused snapshots related to it will be deleted during snapshot update. This resulted in some vicious circle, in which created snapshots for Python 3.12 test deleted the snapshots for 3.11. And then, recreating ones for 3.11 deleted the 3.12.
Skiping at module level solves this.